app/src/main/java/id/rizmaulana/covid19/data/model/CovidOverviewItem.kt

```diff
@@ -8,6 +8,6 @@ import com.google.gson.annotations.SerializedName
  * rizmaulana 04/03/20.
  */
 data class CovidOverviewItem(
-    @Expose @SerializedName("detail") val detail: String? = null,
+    @Expose @SerializedName("detail") val detail: String = "",
```

Comment:
Refer to https://github.com/rizmaulana/kotlin-mvvm-covid19/pull/2#discussion_r390191156